test/blackbox-tests/test-cases/pkg/withenv.t

```diff
@@ -0,0 +1,17 @@
+Setting environment variables in actions
+
+  $ mkdir dune.lock
+  $ cat >dune.lock/lock.dune <<EOF
+  > (lang package 0.1)
+  > EOF
+  $ cat >dune.lock/test <<'EOF'
+  > (build
+  >  (withenv
+  >   ((= FOO BAR)
+  >    (= BAR xxx)
+  >    (+= BAR yyy))
```

Comment:
I'm wondering whether for usability a short form of the type `(withenv (ENV VALUE) ...)` might be useful.

Reply:
Note that this form isn't going to be written by users so I don't mind making it verbose. If we introduce something ergonomic, I would rather move away from the cryptic symbols altogether and stick to names like `append` or `prepend`.